src/Idris/Package.idr

```diff
@@ -558,11 +558,10 @@ processPackageOpts opts
     = do (MkPFR (Just (cmd, f)) opts' err) <- filterPackageOpts (MkPFR Nothing Nil False) opts
              | (MkPFR Nothing opts _) => pure False
 
-         if err
```

Comment:
This gives an error when bootstrapping from Idris2-boot.

Reply:
I reverted the change.